aspnet/conceptual-overview/aspnet.rst

```diff
@@ -1,37 +1,37 @@
 Introduction to ASP.NET Core
 ============================
 
-By `Daniel Roth`_
+By `Daniel Roth`_ and `Rick Anderson`_
 
 ASP.NET Core is a significant redesign of ASP.NET. This topic introduces the new concepts in ASP.NET Core and explains how they help you develop modern web apps.
 
 .. contents:: Sections:
   :local:
   :depth: 1
-  
+
 What is ASP.NET Core?
 ---------------------
 
-ASP.NET Core is a new open-source and cross-platform framework for building modern cloud-based Web applications using .NET. We built it from the ground up to provide an optimized development framework for apps that are either deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core applications cross-platform on Windows, Mac and Linux. ASP.NET Core is fully open source on `GitHub <https://github.com/aspnet/home>`_.
+ASP.NET Core is a new open-source and cross-platform framework for building modern cloud-based Web apps using .NET. We built it from the ground up to provide an optimized development framework for apps that are deployed to the cloud or run on-premises. It consists of modular components with minimal overhead, so you retain flexibility while constructing your solutions. You can develop and run your ASP.NET Core apps cross-platform on Windows, Mac and Linux. ASP.NET Core is open source at `GitHub <https://github.com/aspnet/home>`_.
```

Comment:
Make Web lowercase throughout. http://www.poynter.org/2016/ap-style-change-alert-dont-capitalize-internet-and-web-any-more/404664/